dm/checker/checker.go

```diff
@@ -144,6 +147,25 @@ func (c *Checker) Init(ctx context.Context) (err error) {
 	if egErr := eg.Wait(); egErr != nil {
 		return egErr
 	}
+	// check connections
+	if _, ok := c.checkingItems[config.ConnAmountChecking]; ok {
+		if len(c.stCfgs) > 0 {
+			switch c.stCfgs[0].Mode {
```

Comment:
Why do we just check the first item?

Reply:
`mode` is a global and unique parameter in a task. It's should be the same across all the subtask derived from the same task.

Reply:
Is there any code to ensure this behavior? It's better to leave some comments here to make code clear.

Reply:
yes. will leave a comment here.

FYI: https://github.com/pingcap/tiflow/blob/bd0cc36dd74d12aa7a71a6a6fd7e9d51f14eb242/dm/config/task_converters.go#L49

Reply:
5827f28